runtime/modules/isp_vcs.py

```diff
@@ -221,8 +224,12 @@ def runSim(exe_path, run_dir, policy_dir, pex_path, runtime, rule_cache,
     if isp_utils.generateTagInfo(exe_path, run_dir, policy_dir, soc_cfg=soc_cfg, arch=arch) is False:
         return isp_utils.retVals.TAG_FAIL
 
+    policy = policy_name.replace(debug_suffix, "") if policy_name.endswith(debug_suffix) else policy_name
+    if not shutil.which(f"tag_mem_hexdump-{policy}") and \
+       not installTagMemHexdump(policy_name, run_dir, extra_args.processor):
+        return False
```

Comment:
Maybe you should return isp_utils.NO_BIN, or better yet, add (as part of the isp_utils.py) and failure code the indicate that error.

Reply:
This should be fixed now.